src/main/java/com/ericsson/ei/frontend/utils/BackEndInstanceFileUtils.java

```diff
@@ -22,16 +22,34 @@
 public class BackEndInstanceFileUtils {
 
     private static final Logger LOG = LoggerFactory.getLogger(BackEndInstanceFileUtils.class);
-    private static final String PATH_TO_WRITE = "src/main/resources/EIBackendInstancesInformation.json";
+
+    private static final String BACKEND_INSTANCES_FILENAME = "EIBackendInstancesInformation.json";
+    private static final String EI_HOME_DEFAULT_NAME = ".eiffel";
 
     @Value("${ei.backendInstancesPath:#{null}}")
```

Comment:
As long as the value ei.backendInstancesPath is set in application.properties this will write to src/main/resources/EIBackendInstancesInformation.json 

Maybe you should remove the @Value("${ei.backendInstancesPath:#{null}}") and then the line in application.properties. Since it is replaced with ei.home.

Reply:
backendInstancesPath does not exist anymore, its just called "backendInstancesFileName" and it's just there so that you can choose filename if you would have multiple files in the eiffell home folder for some reason.

ei.home just tells the system where the user configuration folder is placed.

No it won't write to src/main/resources/EIBackendInstancesInformation.json. It will write in the EI home path which by default is user.home/.eiffel.

Reply:
I see that you changed the code since my comment.